src/css/custom.css

```
@@ -419,14 +439,53 @@ div.markdown h3 a:hover {
 
 .mediaImage {
   width: 100%;
-  height: auto
+  height: auto;
 }
 
 /* remove extra vertical space from tabs when it contains a code block */
-.tabs-container .theme-code-block {
-  margin-top: -1.4rem;
+.tabs-container .margin-top--md .theme-code-block {
+  margin-top: -1rem;
+}
+
+/* code title styling */
+div.codeBlockTitle_node_modules-\@docusaurus-theme-classic-lib-theme-CodeBlock-Content-styles-module {
+  font-size: 1.05rem;
+  font-weight: 700;
+  color: var(--ifm-color-gray-300);
+  border-bottom: 3px solid var(--ifm-code-background-color-light);
 }
 
 code span::selection {
   background: var(--docusaurus-highlighted-code-line-bg);
 }
+
+.tabs-container {
+  background-color: var(--ifm-code-background-color);
+  border-radius: var(--ifm-code-border-radius);
+}
+
+.tabs-container .tabs {
+  box-shadow: inset 0px -6px 0px -3px var(--ifm-code-background-color-light);
+}
```

I think it's good to have a subtle contrast between the tabs & the codeblock like we have in prod. 

```suggestion
.tabs-container .tabs {
  background-color: #1f2937;
  border-radius: 0.4rem 0.4rem 0 0;
  box-shadow: inset 0px -6px 0px -4px var(--ifm-code-background-color-light);
}
```